elasticpath/react-pwa-reference-storefront
# Feature/bloomreach integration

Description:


This PR brings in a bloomreach search suggestions component that is dynamically swapped based on the ep.config.json.  The config has been set up to point at a demo bloomreach server that has been set up for us.  It has been loaded with the latest vestri catalog so the suggestions will be correct values.  The component is the exact same as the original search component except that it takes up the entire right side of the header bar (needed more space for suggestions with long names).


Linting:



Component Updates:




Tests:


1.  Enable bloomreach search in ep.config.json
2. Notice that the right search bar will increase in size
3. type characters and watch suggestions appear below search bar
4. navigate the suggestions with your keyboard or click items with mouse
5. notice that search will be correctly made.


NOTE: The new component is not available when app is in mobile mode.  Not sure if we want to include the suggestions in mobile mode as well.  Also this component does not yet have a story for it.  Let me know if you'd like to include both of these things in this PR or of you'd like me to include it in another

** MERGE branch #416 & #417 before merging this one ** 

Documentation:
